python-backend/app/api/permits/permit_amendment/resources/permit_amendment.py

```diff
@@ -134,6 +166,8 @@ def put(self, permit_guid=None, permit_amendment_guid=None):
             else:
                 setattr(pa, key, value)
 
+        pa.save()
```

Comment:
Should this be in the else block or should the save in the if block be removed?

Reply:
Changed